connect/src/com/telenor/connect/id/IdToken.java

```diff
@@ -23,7 +34,16 @@ public IdToken(String serializedSignedJwt) {
                     "Could not parse saved id token. idToken=" + this, e);
         }
 
+        final Map<String, Object> cc = jwtClaimsSet.getCustomClaims();
```

Comment:
Move the declaration closer to the first usage, that is, one line down. Consider avoiding short and cryptic variable names, even in such a limited scope: cc => customClaims.